content/minfraud/working-with-transaction-dispositions.mdx

```diff
@@ -198,18 +198,18 @@ description of the error and may change at any time.
 In addition to the errors documented below, client code should also be prepared
 to handle any valid HTTP 4xx or 5xx status code.
 
-| Code                   | HTTP Status                | Error                                                                                                                                 |
-| ---------------------- | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
-| UPDATES_AFTER_REQUIRED | 400 Bad Request            | You have not supplied the `updates_after`  URI parameter.                                                                             |
-| TIMESTAMP_INVALID      | 400 Bad Request            | The `updates_after` field must be in RFC 3339 format.                                                                                 |
-| PARAMETER_UNKNOWN      | 400 Bad Request            | You have supplied one or more parameters which are not used by this endpoint.                                                         |
-| AUTHORIZATION_INVALID  | 401 Unauthorized           | Your account ID or license key could not be authenticated.                                                                            |
-| LICENSE_KEY_REQUIRED   | 401 Unauthorized           | An account ID and license key are required to use this service.                                                                       |
-| ACCOUNT_ID_REQUIRED    | 401 Unauthorized           | An account ID and license key are required to use this service.                                                                       |
-| PERMISSION_REQUIRED    | 403 Forbidden              | You do not have permission to use the service. Please contact [support@maxmind.com](mailto:support@maxmind.com) for more information. |
-| (none)                 | 406 Not Acceptable         | Your request included an `Accept-Charset` header that is not supported. `UTF-8` is the only acceptable character set.                 |
-| (none)                 | 415 Unsupported Media Type | Your request included an `Accept` header that is not supported. The web service cannot return content of that type.                   |
-| (none)                 | 503 Service Not Available  | There is a problem with the web service server. You can try this request again later.                                                 |
+| Code                   | HTTP Status                | Error                                                                                                                 |
```

Comment:
It seems like we are going to end up having a table very similar to this on a lot of pages, with some variance based on 400 requests. We don't have to abstract this now, but it might be worth starting to think about it.